@navikt/core/css/form/combobox.css

```diff
@@ -102,6 +102,11 @@
   min-width: 10ch;
   width: 100%;
   height: var(--__ac-combobox-input-height);
+  caret-color: transparent;
+}
+
+.navds-combobox__input--show-caret {
+  caret-color: unset;
```

Comment:
Would it be difficult to reverse this? Meaning to use a class to hide the caret instead of hiding it by default and then unsetting that style when showing it (which is used in all cases except for after selecting an option).

Reply:
no, can absolutely reverse it. I had that at first with `hideCaret`, but went for this mainly due to confusion with "hide" being negative 😅 , thinking boolean variables should always "stay positive" in their naming scheme by default, but thinking about it now, it's more about not having `notHidden` or `willNot...` etc. as a variable. https://www.serendipidata.com/posts/naming-guidelines-for-boolean-variables

tl;dr, i'll change it to `--hide-caret` / `hideCaret` with a single CSS rule.